docs/en_US/deploy-locally.md

```diff
@@ -0,0 +1,207 @@
+# Deploy ob-operator locally
+
+## Hardware requirements
+
+* CPU: 4 Core available
+* Memory: 10 GB available (A Single-node OceanBase cluster takes 8 ~ 9 GB memory)
+* Disk space: 120 GB available
+
+## Software requirements
+
+You should install following two software before we start. Installation instructions could be found on their sites. You can get minkube by brew install minikube if you develop on macOS.
+* [kubernetes/minikube](https://github.com/kubernetes/minikube): Run kubernetes locally
+* [Docker desktop](https://www.docker.com/products/docker-desktop/): Manage container development locally
```

Comment:
development -> deployments ?

Reply:
It's not a typo. What I mean is writing dockerfile, building and running images, manage manifests and so on. Not only deployments.